content/blogposts/blog-post-starter-pack.md

```diff
@@ -0,0 +1,271 @@
+  alt="Sourcegraph thumbnail"
+  caption="The Sourcegraph logo"
+/>
+
+
+### TableWrapper
+
+Tables are a straightforward way to present and compare data. When preparing a table, please include the `TableWrapper` component to ensure that the table adapts to different screen sizes. To support your preference, our blog accepts tables written in both Markdown and HTML.
+
+This table is written in **Markdown**:
+
+<TableWrapper>
+| Id   | Species     | Sepal Length (cm) | Sepal Width (cm) | Petal Length (cm) | Petal Width (cm) |
+|------|-------------|-------------------|------------------|-------------------|------------------|
+| 1    | Iris Setosa | 5.1               | 3.5              | 1.4               | 0.2              |
+| 2    | Iris Setosa | 4.9               | 3.0              | 1.4               | 0.2              |
+</TableWrapper>
+
+This is the same table, but written in **HTML**:
+
+<TableWrapper>
+  <table>
+    <thead>
+      <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Sepal Length (cm)</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>1</td>
+        <td>Iris Setosa</td>
+        <td>5.1</td>
+        <td>3.5</td>
+        <td>1.4</td>
+        <td>0.2</td>
+      </tr>
+      <tr>
+        <td>2</td>
+        <td>Iris Setosa</td>
+        <td>4.9</td>
+        <td>3.0</td>
+        <td>1.4</td>
+        <td>0.2</td>
+      </tr>
+    </tbody>
+  </table>
+</TableWrapper>
+
+
+### YouTube
+
+You can easily embed YouTube videos using the `YouTube` component. The two required properties are `title` and `id`.
+
+A title should be descriptive for your YouTube video and is used for accessibility and the caption. It is also hidden by default. To optionally display the title below the video as a caption, add the `showTitle` property.
+
+The `id` value is found at the end of the URL for any YouTube video.
+
+<YouTube
+  title="Accelerate developer onboarding with Sourcegraph" 
+  id="DgwvhRW1Cbc"
+  showTitle={true}
+/>
+
+
+### Video
+
+The required properties for the `Video` component are `mp4`, `webm`, `loop`, `caption`, and `title`. 
+
+Since the component supports both `.mp4` and `.webm` files, you'll need to upload the two versions of the video to our Google Cloud [assets bucket](https://console.cloud.google.com/storage/browser/sourcegraph-assets/blog) under an appropriate sub directory. Then, add the file path from the Public URL of these files to the `mp4` and `webm` properties without their file extensions.
+
+You can choose to repeat a video by including the `loop` property, optionally auto play the video by including the `autoPlay` property, or optionally display the caption using the `showCaption` property.
+
+<Video 
+  source={{webm: 'blog/shift-left/01-code-reuse', mp4: 'blog/shift-left/01-code-reuse'}} 
+  loop={true}
+  title="Code reuse"
+  caption="An example of code search with Sourcegraph"
+  showCaption={true}
+/>
```

Comment:
This will cause the build to break though. When #5615 is merged I'll commit this. Or if this goes in first, #5615 can make this change to the file.

Reply:
@st0nebraker Yes I know, that's what I'm saying - we should wait for #5615 to go in first since we don't want to encourage people to write this code since it will break.

Reply:
Can you break the bug fix into a separate PR to separate concerns? @bretthayes

Reply:
@st0nebraker I updated the language slightly in the suggestion and reverted back to the original syntax since I confirmed that it's working in #5619